crypto/keyring/keyring_test.go

```diff
@@ -788,41 +954,65 @@ func ExampleNew() {
 	// signed by Bob
 }
 
+func getIndices(list []*Record, uid1, uid2, uid3 string) (i, j, k int) {
```

Comment:
`getIndices()` is used to fetch indices that are used in `TestAltKeyring_List`

Reply:
This indices manipulation is really confusing IMO. Is there some way to return in some deterministic order (like alphabetical like before)?

Reply:
I came up with an idea to use  `func getIndices(list []*Record, uids ...string) map[string]int ` to return a `hashmap[uid] = index` .  You can find more details [here](https://github.com/cosmos/cosmos-sdk/blob/cyberbono3/keyring-migration/crypto/keyring/keyring_test.go#L76). Does it sounds good to you?
```


```